cfgov/unprocessed/js/organisms/FilterableListControls.js

```diff
@@ -275,5 +144,7 @@ function FilterableListControls(element) {
 }
 
 FilterableListControls.BASE_CLASS = BASE_CLASS;
```

Comment:
```suggestion
```

It seems like this line is no longer needed? It was previously used [here](https://github.com/cfpb/consumerfinance.gov/blob/defb5e59415a10a1ec176a47f15a1cb4c8925199/cfgov/unprocessed/js/organisms/FilterableList.js#L35).

Reply:
This follows our general component patterns, so I think we should keep it to keep the component boilerplate the same:
https://github.com/cfpb/design-system/blob/main/packages/cfpb-forms/src/organisms/Multiselect.js#L578

This just allows retrieving the CSS class name from the component without needing to hardcode the CSS class name outside of the component.